### PR TITLE
parsplice

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/lammps-feedstock/pull/32 to be merged.

File: recipes/parsplice/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mkdir build
+cd build
+cmake -DCMAKE_PREFIX_PATH="${PREFIX}"  -DCMAKE_INSTALL_PREFIX="${PREFIX}" ../
```
Comment (marcelotrevisani) [inline recipes/parsplice/build.sh]: Did you forget to run ``make``?
```suggestion
cmake -DCMAKE_PREFIX_PATH="${PREFIX}"  -DCMAKE_INSTALL_PREFIX="${PREFIX}" ../ 
make -j${CPU_COUNT}
```

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/libdb-feedstock/pull/7

File: recipes/parsplice/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - boost
```
Comment (bgruening) [inline recipes/parsplice/meta.yaml]: is boost-cpp enough or do you need the python bindings as well?

**Comment by jan-janssen** [comment]
Waiting for https://github.com/conda-forge/libdb-feedstock/pull/8 to be merged.

**Comment by jan-janssen** [comment]
While dblib is fixed we are now missing: `nauty/traces.h` so that is the next thing to look at.

**Comment by jan-janssen** [comment]
@conda-forge/help-c-cpp Somehow I have some issue with the library path once the executable is moved to the final location.

**Comment by jan-janssen** [comment]
There seems to be something wrong with the path:
```
Warning: rpath /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
Warning: rpath /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
Warning: rpath /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
Warning: rpath /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/parsplice_1576039214069/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
  ERROR :: $RPATH/liblammps.so.0 not in prefix_owners
  ERROR :: lib/libdb_cxx-6.1.so not in prefix_owners
  ERROR :: $RPATH/liblammps.so.0 not in prefix_owners
  ERROR :: lib/libdb_cxx-6.1.so not in prefix_owners
  ERROR :: $RPATH/liblammps.so.0 not in prefix_owners
  ERROR :: lib/libdb_cxx-6.1.so not in prefix_owners
  ERROR :: $RPATH/liblammps.so.0 not in prefix_owners
  ERROR :: lib/libdb_cxx-6.1.so not in prefix_owners
   INFO (parsplice,bin/dbextract): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libpthread.so.0 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/dbextract): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libc.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/dbextract): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/dbextract): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libm.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/dbextract): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/dbextract): Needed DSO lib/libmpi.so.12 found in conda-forge::mpich-3.3.2-hc856adb_0
   INFO (parsplice,bin/dbextract): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
  ERROR (parsplice,bin/dbextract): $RPATH/liblammps.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (parsplice,bin/dbextract): Needed DSO lib/libdb_cxx-6.1.so found in conda-forge::libdb-6.1.26-he1b5a44_2001
   INFO (parsplice,bin/parsplice): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libpthread.so.0 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libc.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libm.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice): Needed DSO lib/libmpi.so.12 found in conda-forge::mpich-3.3.2-hc856adb_0
   INFO (parsplice,bin/parsplice): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
  ERROR (parsplice,bin/parsplice): $RPATH/liblammps.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (parsplice,bin/parsplice): Needed DSO lib/libdb_cxx-6.1.so found in conda-forge::libdb-6.1.26-he1b5a44_2001
   INFO (parsplice,bin/parsplice-label): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libpthread.so.0 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-label): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libc.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-label): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice-label): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libm.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-label): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice-label): Needed DSO lib/libmpi.so.12 found in conda-forge::mpich-3.3.2-hc856adb_0
   INFO (parsplice,bin/parsplice-label): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
  ERROR (parsplice,bin/parsplice-label): $RPATH/liblammps.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (parsplice,bin/parsplice-label): Needed DSO lib/libdb_cxx-6.1.so found in conda-forge::libdb-6.1.26-he1b5a44_2001
   INFO (parsplice,bin/parsplice-md): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libpthread.so.0 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-md): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libc.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-md): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice-md): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/libm.so.6 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
   INFO (parsplice,bin/parsplice-md): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-9.2.0-hdf63c60_0
   INFO (parsplice,bin/parsplice-md): Needed DSO lib/libmpi.so.12 found in conda-forge::mpich-3.3.2-hc856adb_0
   INFO (parsplice,bin/parsplice-md): Needed DSO x86_64-conda_cos6-linux-gnu/sysroot/lib/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge::gcc_impl_linux-64-7.3.0-habb00fd_2
  ERROR (parsplice,bin/parsplice-md): $RPATH/liblammps.so.0 not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (parsplice,bin/parsplice-md): Needed DSO lib/libdb_cxx-6.1.so found in conda-forge::libdb-6.1.26-he1b5a44_2001
WARNING (parsplice): dso library package conda-forge::boost-cpp-1.70.0-h8e57a91_2 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (parsplice): dso library package conda-forge::lammps-2019.08.07-py37_mpich_5 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```
Or is this an issue because the code tries to link using relative paths?

**Comment by jan-janssen** [comment]
@isuruf Can you help me with this issue?

**Comment by jan-janssen** [comment]
New pull request ready to merge under https://github.com/conda-forge/staged-recipes/pull/10675

**Pull request closed without merging**